src/mawpy/workflows/ao_ic_usd.py

```diff
@@ -40,7 +40,7 @@ def ao_ic_usd(
     return df_output_final
 
 
-if __name__ == "__main__":
+def main():
```

Comment:
Just realized that the docstrings for `main` should be in each of the workflow function of `ao_ic_usd` for example. And that the jupyterbook rst should be pointing to that function instead of `main`